libs/utils/src/main/java/com/akto/data_actor/DataActor.java

```diff
@@ -235,4 +235,6 @@ public abstract class DataActor {
     public abstract List<YamlTemplate> fetchFilterYamlTemplates();
 
     public abstract void insertProtectionLog(Log log);
+    
+    public abstract List<YamlTemplate> fetchActiveAdvancedFilters();
```

Comment:
Since this is an abstract class, all implementations of the abstract class must implement all the functions of the same. **THIS CODE WILL NOT COMPILE.** Please run the code once, before submitting.

Reply:
It's compiling, refresh once
and dont take assumption please before commenting

Reply:
<img width="1008" alt="Screenshot 2024-09-16 at 9 37 40 AM" src="https://github.com/user-attachments/assets/302b9a2b-140d-4e5d-910d-10d8c9e9f574">

well, it was not when I reviewed, not assuming here.